pipenv/installers.py

```diff
@@ -117,7 +117,10 @@ def _run(self, *args, **kwargs):
             k = list(kwargs.keys())[0]
             raise TypeError(f'unexpected keyword argument {k!r}')
         args = (self.cmd,) + tuple(args)
-        c = subprocess_run(args, timeout=timeout)
+        if os.name == 'nt':
```

Comment:
I don't think this is necessary.   I've tested the current behavior of `c = subprocess_run(args, timeout=timeout)` on classic windows cmd, power shell and git shell and it does not seem to error out.